### ISSUE TITLE
md-select fails chrome accessibility dev tools audit

### ISSUE THREAD

**Issue by opadillaintegrify** [angular/material#4914]
Assuming that https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=en is installed...

Go to https://material.angularjs.org/0.11.1/#/demo/material.components.select, open dev tools, go to audit tab and run an audit on that page.

Notice that every instance of md-select throws:
[Severe] Elements with ARIA roles must ensure required owned elements are present (6)

Perhaps it's expecting the aria roles to look more like this:
http://www.w3.org/TR/wai-aria/roles#combobox

**Comment by marcysutton** [comment]
This could be fixed along with https://github.com/angular/material/issues/3859. There are also failures for duplicate IDs in SVG icons, specifically the menu icon.

**Comment by calebegg** [comment]
Note that the duplicate ID check triggering on SVGs should be fixed:

https://github.com/GoogleChrome/accessibility-developer-tools/issues/141

But it's possible that the Chrome store version of the extension hasn't been updated.
